_includes/head.html

```diff
@@ -42,19 +42,18 @@
   {% endif %}
 
   <!-- Icons -->
-  <link rel="apple-touch-icon" sizes="57x57" href="{{ "/assets/icons/apple-touch-icon-57x57.png" | relative_url }}">
-  <link rel="apple-touch-icon" sizes="114x114" href="{{ "/assets/icons/apple-touch-icon-114x114.png" | relative_url }}">
-  <link rel="apple-touch-icon" sizes="72x72" href="{{ "/assets/icons/apple-touch-icon-72x72.png" | relative_url }}">
-  <link rel="apple-touch-icon" sizes="144x144" href="{{ "/assets/icons/apple-touch-icon-144x144.png" | relative_url }}">
-  <link rel="apple-touch-icon" sizes="60x60" href="{{ "/assets/icons/apple-touch-icon-60x60.png" | relative_url }}">
-  <link rel="apple-touch-icon" sizes="120x120" href="{{ "/assets/icons/apple-touch-icon-120x120.png" | relative_url }}">
-  <link rel="apple-touch-icon" sizes="76x76" href="{{ "/assets/icons/apple-touch-icon-76x76.png" | relative_url }}">
-  <link rel="apple-touch-icon" sizes="152x152" href="{{ "/assets/icons/apple-touch-icon-152x152.png" | relative_url }}">
-  <link rel="apple-touch-icon" sizes="180x180" href="{{ "/assets/icons/apple-touch-icon-180x180.png" | relative_url }}">
-  <link rel="icon" type="image/png" href="{{ "/assets/icons/favicon-192x192.png" | relative_url }}" sizes="192x192">
-  <link rel="icon" type="image/png" href="{{ "/assets/icons/favicon-160x160.png" | relative_url }}" sizes="160x160">
-  <link rel="icon" type="image/png" href="{{ "/assets/icons/favicon-96x96.png" | relative_url }}" sizes="96x96">
-  <link rel="icon" type="image/png" href="{{ "/assets/icons/favicon-16x16.png" | relative_url }}" sizes="16x16">
-  <link rel="icon" type="image/png" href="{{ "/assets/icons/favicon-32x32.png" | relative_url }}" sizes="32x32">
+  {% for static_file in site.static_files -%}
+    {% if static_file.path contains "/assets/icons/apple-touch-icon-" or static_file.path contains "/assets/icons/favicon-" -%}
+      {% assign path_components = static_file.path | split: "/" -%}
+      {% assign icon = path_components[-1] | split: "-" -%}
+      {% if icon[0] == "apple" -%}
+        {% assign rel = "apple-touch-icon" -%}
+      {% else -%}
+        {% assign rel = "favicon" -%}
```

Comment:
`rel` type for favicons [should be `icon`](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel#icon).

Reply:
Fixed.